<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Compute SPAREs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>338</width>
           <height>241</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="show_SmileGAN_prob_from_data_Btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Show SmileGAN patterns from data</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="load_SmileGAN_model_Btn">
         <property name="text">
          <string>Load SmileGAN Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="SmileGAN_model_info">
         <property name="text">
          <string>No SmileGAN model loaded</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="compute_SmileGAN_Btn">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Compute SmileGAN patterns</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>338</width>
           <height>241</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>SmileGAN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="load_other_model_Btn">
           <property name="text">
            <string>Load other model</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="add_to_dataframe_Btn">
           <property name="text">
            <string>Add to DataFrame</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
